### DISCUSSION TITLE
Replace  Internal Radio firmware with ELRS firmware on for Frsky X-lite.

### DISCUSSION THREAD

**Discussion by sincomil** [ExpressLRS/ExpressLRS#2173] [Q&A]
Is it possible to flash ELRS firmware replacing internal module firmware of FrSky Tataranis X-Lite (S/Pro) without any modification of board electronics? This can save external module functionality of this radio.

**Comment by itsmerandy** [comment]
https://youtu.be/8ZfG6l0pAVU
